plasTeX/Imagers/__init__.py

```diff
@@ -636,12 +638,17 @@ def compileLatex(self, source):
                         cmd_line))
         output = None
         for ext in ['.dvi','.pdf','.ps']:
-            if os.path.isfile('images'+ext):
-                output = open('images'+ext, 'rb')
+            try:
+                with open('images'+ext, 'rb') as f:
+                    output = open('images'+ext, 'rb').read()
```

Comment:
This is a pretty risky change. Did you triple checked you propagated it everywhere?

Reply:
Yeah. I grep'ed the source and made sure it is only propagated to `executeConvert`, and I modified all the `executeConvert`'s.